README.md

```diff
@@ -1289,6 +1289,27 @@ Other Style Guides
 
   - [15.4](#15.4) <a name='15.4'></a> For more information see [Truth Equality and JavaScript](http://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/#more-2108) by Angus Croll.
 
+  - [15.5](#15.5) <a name='15.5'></a> Ternaries should not be nested and generally be single line expressions.
```

Comment:
How about something like:
> - Ternaries should not be nested.
> - Single line ternaries should be preferred, unless the resulting line is _long_.
>
>     ```js
>     // good 
>     const val = _.isFunction(x.foo) ? x.foo() : x.backupFoo();
>     // good
>     const name = 'nickname' in this
>         ? `${this.firstName} "${this.nickname}" ${this.surname}`
>         : `${this.firstName} ${this.surname}`;
>     ```

In my opinion "_long_" should be "> 80 characters", but this style guide suggests 100 elsewhere.